nominal/core/stream.py

```diff
@@ -19,31 +21,38 @@ class BatchItem:
     channel_name: str
     timestamp: str | datetime | IntegralNanosecondsUTC
     value: float
-    tags: Dict[str, str] | None = None
+    tags: dict[str, str] | None = None
+
 
+NominalWriteStream: TypeAlias = "WriteStream"
```

Comment:
This should be actually equal to the type, not just a type alias

Reply:
alternatively we could also deprecate this via the module's `__getattr__`

Reply:
The user should not create an instance directly, but use `connection.get_nominal_write_stream` which already has a deprecation warning.